controllers/books.controller.js

```diff
@@ -30,23 +45,34 @@ export const BooksController = {
 
 	async updateBook(req, res, next) {
 		try {
+			const userId = req.body.userId; //should I get it from session l8r?
+			let updateBookOptions;
+			if(userId) {					//if userID was provided, we book the book for this user
```

Comment:
you miss empty line between logically different sections of code.

Reply:
empty line added